### ISSUE TITLE
Include jq language specification in jqlang

### ISSUE THREAD

**Issue by 01mf02** [jqlang/jq#3182]
For the last months, I have been writing a [formal specification of the jq language](https://github.com/01mf02/jq-lang-spec). In https://github.com/01mf02/jq-lang-spec/issues/1, @wader and @nicowilliams were sympathetic to the idea to make this repository part of [the jqlang organisation](https://github.com/jqlang). I think that the specification has now reached a point where such an inclusion could happen.
This step would benefit my work by increasing its visibility, and IMHO also benefit jqlang by featuring a formal specification.

I am also happy to accept contributions from other maintainers to augment the specification; for example, I have consulted with @itchyny on the topic of [path-based update semantics](https://github.com/01mf02/jaq/issues/139#issuecomment-2326344007_), but did not document this in the specification yet.

TL;DR: Would you like to include my jq language specification into jqlang?

**Comment by chb0github** [comment]
I actually opened a [ticket with jetbrains ](https://youtrack.jetbrains.com/issue/IJPL-149996/Support-for-jq-lang) and one of the things they asked/commented on was a language spec. This certainly would help

**Comment by 01mf02** [comment]
Thanks for the link, @chb0github! I would really like to see my work being adopted by the jq organisation, but unfortunately, there is very little movement going here.
@pkoppstein, @wader, @nicowilliams, @itchyny, is there any news on this topic?
